src/db/oracledb/jobs-dao.js

```diff
@@ -56,4 +120,269 @@ const getJobByJobId = async (internalId, jobId) => {
+ * @param {object} body Request body
+ * @returns {string} Query result, null if success
+ */
+const updateLaborChangeJob = async (connection, osuId, body) => {
+  const binds = standardBinds(osuId, body, ['laborDistribution']);
+
+  const { outBinds: { result } } = await connection.execute(contrib.updateLaborChangeJob(), binds);
+  return result;
+};
+
+/**
+ * Updates student job records
+ *
+ * @param {object} connection oracledb connection
+ * @param {string} osuId OSU ID of a person
+ * @param {object} body Request body
+ * @returns {string} Query result, null if success
+ */
+const studentUpdateJob = async (connection, osuId, body) => {
+  const binds = standardBinds(osuId, body, studentBinds);
+
+  const result = await connection.execute(contrib.studentUpdateJob(binds), binds);
+  return result.outBinds.result;
+};
+
+/**
+ * Creates student job records
+ *
+ * @param {object} connection oracledb connection
+ * @param {string} osuId OSU ID of a person
+ * @param {object} body Request body
+ * @returns {string} Query result, null if success
+ */
+const studentCreateJob = async (connection, osuId, body) => {
+  const binds = standardBinds(osuId, body, studentBinds);
+
+  const { outBinds: { result } } = await connection.execute(contrib.studentCreateJob(binds), binds);
+  return result;
+};
+
+/**
+ * Update or create graduate job record
+ *
+ * @param {object} connection oracledb connection
+ * @param {string} osuId OSU ID of a person
+ * @param {object} body Request body
+ * @param {string} update 'create' to create a new record, 'update' to update an existing one
+ * @returns {string} Query result, null if success
+ */
+const graduateJob = async (connection, osuId, body, update) => {
+  const binds = standardBinds(osuId, body, graduateBinds);
+  const queryName = update === 'update' ? 'graduateUpdateJob' : 'graduateCreateJob';
+
+  const { outBinds: { result } } = await connection.execute(
+    contrib[queryName](binds),
+    binds,
+  );
+  return result;
+};
+
+/**
+ * Determines if changeReasonCode is valid
+ *
+ * @param {object} connection oracledb connection
+ * @param {string} changeReasonCode Change reason code from request body
+ * @returns {boolean} True if changeReasonCode is valid
+ */
+const isValidChangeReasonCode = async (connection, changeReasonCode) => {
+  const { rows } = await connection.execute(
+    contrib.validateChangeReasonCode(),
+    { changeReasonCode },
+  );
+  return rows[0].count > 0;
+};
+
+/**
+ * Determines which Epaf to execute based on fields in body
+ *
+ * @param {string} update 'create' to create a new record, 'update' to update an existing one
+ * @param {string} osuId OSU ID of a person
+ * @param {object} body Request body
+ * @param {string} internalId Internal ID of a person
+ * @returns {object} raw job record
+ */
+const createOrUpdateJob = async (update, osuId, body) => {
+  const connection = await getConnection('banner');
+  try {
+    let result;
```

Comment:
Sounds like this is an error for the result so I recommend renaming it as `error` or something similar.